src/addrman.h

```diff
@@ -404,15 +404,15 @@ class CAddrMan
     {
         AssertLockHeld(cs);
 
-        const int err = Check_();
+        const int err{CheckAddrMan()};
         if (err) {
             LogPrintf("ADDRMAN CONSISTENCY CHECK FAILED!!! err=%i\n", err);
             assert(false);
         }
     }
 
     //! Perform consistency check. Returns an error code or zero.
-    int Check_() const EXCLUSIVE_LOCKS_REQUIRED(cs);
+    int CheckAddrMan() const EXCLUSIVE_LOCKS_REQUIRED(cs);
```

Comment:
any reason for the rename here? This will just cause conflicts and increase the git history without any benefit.

Reply:
It's to have the log print "CheckAddrMan:" instead of "Check_:"

Reply:
All the function names seem to end with `_`, might want to keep that at least for consistency?

Reply:
Done, updated the pull description